src/backend/efiling-reviewer-api/README.md

```diff
@@ -25,7 +25,7 @@ You should use environment variables to configure the jag efiling api
 
 | Environment Variable            | Type    | Description                                  | Notes                          |
 | ------------------------------- | ------- | -------------------------------------------- | ------------------------------ |
-| SERVER_PORT                     | Integer | web application server port                  | defaulted to `8080`            |
+| SERVER_PORT                     | Integer | web application server port                  | defaulted to `8090`            |
```

Comment:
It is actually defaulted to 8080 in the config, but set to 8090 in docker.

Reply:
Right.  Should we try to be consistent from compose?  Just thinking that it if I brought up everything in docker-compose, then took down reviewer-api to work on it and had a local instance running of just it, by default it should run on the same port.  If I don't manually configure the port when bringing it up, then it'll fail since something else is already running on port 8080.  Just a thought.

Reply:
It should be ok it maps to port 8090 externally. But it may be worth noting that somewhere.